mods/ra/maps/sarin-gas-3-controlled-burn/controlledburn.lua

```diff
@@ -99,6 +107,30 @@ PowerDownTeslas = function()
 	end
 end
 
+PrepareFinishingHunt = function(player)
+	local buildings = GetBaseBuildings(player)
+
+	Trigger.OnAllKilledOrCaptured(buildings, function()
+		Utils.Do(player.GetGroundAttackers(), function(actor)
+			actor.Hunt()
+		end)
+
+		local harvesters = player.GetActorsByType("harv")
+		Utils.Do(harvesters, function(harvester)
+			-- Skip idle trucks for USSR's required units check.
+			harvester.Owner = BadGuy
```

Comment:
It would be better if we simply removed the requirement to be destroyed from harvesters.

Reply:
I'll remove that in a more straightforward way.